.github/workflows/python-ci.yml

```diff
@@ -20,6 +20,10 @@ jobs:
       env:
         PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
       run: |
+        sudo apt-get install -y curl ca-certificates gnupg
+	sudo curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+	sudo echo 'deb http://apt.postgresql.org/pub/repos/apt focal-pgdg main' | sudo tee /etc/apt/sources.list.d/pgdg.list
```

Comment:
The indentation before `sudo curl` and `sudo echo` is tabs; it should be spaces.